src/plot_topoplotseries.jl

```diff
@@ -32,54 +35,88 @@ Multiple miniature topoplots in regular distances.
     Enable interactive mode. \\ 
     If you create `obs_tuple = Observable((0, 0, 0))` and pass it into `interactive_scatter` you can change observable indecies by clicking topopplot markers.\\
     `(0, 0, 0)` corresponds to the indecies of row of topoplot layout, column of topoplot layout and channell. 
+- `mapping.layout = nothing`\\
+    When equals `:time` arrange topoplots by rows. 
+
 
 $(_docstring(:topoplotseries))
 
 **Return Value:** `Figure` displaying the Topoplot series.
 """
-plot_topoplotseries(data::DataFrame, Δbin::Real; kwargs...) =
-    plot_topoplotseries!(Figure(), data, Δbin; kwargs...)
+plot_topoplotseries(data::DataFrame; kwargs...) =
+    plot_topoplotseries!(Figure(), data; kwargs...)
 
 function plot_topoplotseries!(
     f::Union{GridPosition,GridLayout,Figure,GridLayoutBase.GridSubposition},
-    data::Union{<:Observable{<:DataFrame},DataFrame},
-    Δbin;
+    data::Union{<:Observable{<:DataFrame},DataFrame};
+    bin_width = nothing,
+    bin_num = nothing,
     positions = nothing,
-    labels = nothing,
+    nrows = 1,
+    labels = nothing, # rename to channel_labels?
     combinefun = mean,
-    col_labels = true,
+    col_labels = false,
     row_labels = true,
     rasterize_heatmaps = true,
     interactive_scatter = nothing,
     kwargs...,
 )
 
     data = _as_observable(data)
+    positions = get_topo_positions(; positions = positions, labels = labels)
+    chan_or_label = "label" ∉ names(to_value(data)) ? :channel : :label
 
     config = PlotConfig(:topoplotseries)
     # overwrite all defaults by user specified values
     config_kwargs!(config; kwargs...)
 
     # resolve columns with data
     config.mapping = resolve_mappings(to_value(data), config.mapping)
-
     cat_or_cont_columns =
         eltype(to_value(data)[!, config.mapping.col]) <: Number ? "cont" : "cat"
+    data = deepcopy(to_value(data))
```

Comment:
Why do we copy the data here?

Reply:
I don't remember I put this line. 
Should I just change it to: data = to_value(data)?

Reply:
I dont know

Reply:
i will keep it, otherwise it crashes